src/services/players/player.cpp

```diff
@@ -133,7 +133,7 @@ namespace big
 
 	uint8_t player::id() const
 	{
-		if (*g_pointers->m_gta.m_is_session_started)
+		if (gta_util::get_network_player_mgr()->m_player_count > 0)
```

Comment:
This is a _possible_ nullptr deref crash waiting to happen, however I'm pretty sure the CNetworkPlayerMgr always exists (never deallocated and instantiated once).